model/perf.go

```diff
@@ -637,3 +638,151 @@ func (r *PerformanceResults) FindOutdatedRollups(ctx context.Context, name strin
 
 	return errors.WithStack(it.Close(ctx))
 }
+
+func GetTimeSeriesIds(ctx context.Context, db *mongo.Database) (*mongo.Cursor, error) {
+	pipeline := bson.A{
```

Comment:
I've never used a bson.A, I think a []bson.M{} does the same thing (which you use below,) I'd say stay consistent here.